CHANGELOG.md

```
@@ -0,0 +1,115 @@
+# Changelog
+
+All notable changes to Ghost Story Factory will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [v0.1.0] - 2025-10-25
+
+### Added
+
+**Core Game Engine (Phase 1-2)**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)